content/updates/kickstart-2018.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Kickstart 2018
```

Comment:
While I can see why this name was chosen, it looks a bit odd when combined with the name of the January update ("January 2018"):

![image](https://user-images.githubusercontent.com/336212/36078094-65af2ac6-0f6a-11e8-9f80-47378550b55a.png)

Unfortunately I can't immediately think of a name which is obviously better, though some possible names are "December 2017"; (just) "Kickstart"; "2018 Kickstart"; "Kickstart SB2018"; "SB2018 Kickstart".

Reply:
The more I look at this the more I'm leaning towards "2018 Kickstart".